server/src/resources/js/__mocks__/jquery.js

```diff
@@ -14,6 +14,9 @@ const { parse } = require('graphql');
 // Fill this in with the schema string
 const schemaString = directives + mainSchema + userSchema + productSchema + tagSchema + ordersSchema;
```

Comment:
@AlexeiaW shall we try out the shared schema that's now generated in the root. You can see how this is implemented in the vue client. I'd be interested to see if you current test suite passes with the generated schema.

Ref: https://github.com/gernzy/gernzy/blob/master/client/admin/tests/unit/helper.js#L7